.goreleaser.yml

```diff
@@ -14,7 +15,8 @@ builds:
     - arm64
 
 archives:
-- replacements:
+-
```

Comment:
Is this correct?

Reply:
Yes its correct. `archives` is now an array.